helios-testing/src/main/java/com/spotify/helios/testing/HeliosDeployment.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2015 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.helios.testing;
+
+import com.spotify.helios.client.HeliosClient;
+
+public interface HeliosDeployment extends AutoCloseable {
```

Comment:
probably sounds like a nit, but I think even a one-sentence javadoc summary of the class would be useful